jetstream/src/jsmsg.ts

```diff
@@ -171,8 +171,8 @@ export function parseInfo(s: string): DeliveryInfo {
   di.account_hash = tokens[3];
```

Comment:
Comment on lines 156 and 167 should be updated too?
Also, seem like there is no .random in the end?
```
[15523] 2025/01/08 12:07:23.854376 [TRC] 127.0.0.1:49960 - wid:31 - <<- [PUB $JS.ACK.OCB2058BMO3S4M3H1PCXP2.OCB2058BMO3S4M3H1PDB10_1.1.98.98.1736366843626007307.2 4]   
[15523] 2025/01/08 12:07:23.854411 [TRC] 127.0.0.1:49960 - wid:31 - <<- MSG_PAYLOAD: ["+ACK"]     
```

Reply:
just fixed :)